src/index.css

```diff
@@ -30,17 +49,220 @@ nav ul {
 
 nav ul li {
   cursor: pointer;
+  padding-left: 5%;
   list-style: none;
 }
 
 .active-link {
   text-decoration: none;
+  color: black;
+}
+
+.logo {
+  color: #0590ff;
+}
+
+.user-image {
+  padding-left: 52%;
+  width: 40px;
+  height: 40px;
+}
+
+.each-item {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  justify-content: center;
+  align-content: center;
+  width: 50%;
+  height: 100%;
+  gap: 4px;
+  padding: 0 5% 0 5%;
+}
+
+.each-book-table {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  width: 90%;
+  height: 8%;
+}
+
+.book-category {
+  font-size: 14px;
+  color: #bcbcbc;
+}
+
+.book-title {
+  color: #383838;
+  font-size: 28px;
+}
+
+.book-author {
+  color: #8dbce1;
+  font-size: 16px;
+}
+
+.book-button-holder {
+  display: flex;
+  gap: 10px;
+  padding-top: 3%;
+}
+
+.book-button {
+  color: #8dbce1;
+  padding: 0;
+  border: none;
+  background: none;
+}
+
+.inside-navbar {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+}
+
+.box {
+  display: flex;
+  align-content: center;
+  justify-content: center;
+  align-items: center;
+}
+
+.box .percent {
+  position: relative;
+  width: 150px;
+  height: 150px;
+}
+
+.box .percent svg {
+  position: relative;
+  width: 150px;
+  height: 150px;
+}
+
+.box .percent svg circle {
+  width: 150px;
+  height: 150px;
+  fill: none;
+  stroke-width: 10;
+  stroke: #000;
+  transform: translate(5px, 5px);
+  stroke-dasharray: 440;
+  stroke-dashoffset: 440;
+  stroke-linecap: round;
+}
+
+.box .percent .number {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #111;
+}
+
+.box .percent .number h2 {
+  font-size: 48px;
+}
+
+.box .percent .number h2 span {
+  font-size: 24px;
+}
+
+.box .text {
+  padding: 10px 0 0;
+  color: #999;
+  font-weight: 700;
+  letter-spacing: 1px;
+}
+
+.box .percent svg circle:nth-child(1) {
+  stroke-dashoffset: 0;
+  stroke: #f3f3f3;
+}
+
+.box .percent svg circle:nth-child(2) {
+  stroke-dashoffset: calc(440 - (440 * 60) / 100);
+  stroke: #03a9f4;
+}
+
+.item-all-table {
+  display: flex;
+  width: 100%;
+  height: 14rem;
+  flex-direction: row;
+  background-color: white;
+}
+
+.current-chapter-div {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  align-items: center;
+  padding-left: 6%;
+  gap: 0;
+}
+
+.current-chapter {
+  font-size: 12px;
+  color: #cecece;
+}
+
+.chapter-count {
+  font-size: 16px;
+}
+
+.chapter-button {
+  display: flex;
+  align-content: center;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  width: 10rem;
+  background-color: #0290ff;
   color: white;
+  border: 0.1px solid black;
+  padding: 0.4rem 0.4rem 0.4rem 0.4rem;
```

Comment:
- [ ] Please let's try to make our buttons look exactly like the Zeplin design buttons as shown below. Please remove the black border around the button and add a little bit of padding to get a design that looks closer to the provided one.
### Your Design
![image](https://user-images.githubusercontent.com/46839548/208251028-b92c36df-011e-47f2-a204-de155a57afcc.png)

### Zeplin Design
![image](https://user-images.githubusercontent.com/46839548/208251036-a039b6b2-92d8-4e25-bc80-af055fc6c5e1.png)